website/content/docs/configuration/listener/tcp.mdx

```diff
@@ -205,6 +223,38 @@ listener "tcp" {
 ```
 
 
+### Configuring custom http response headers
+
+This example shows configuring custom http response headers. Operators can configure
+`"custom_response_headers"` sub-stanza in the listener stanza to set custom http
+headers appropriate to their applications. Examples of such headers are `"Strict-Transport-Security"`
+and `"Content-Security-Policy"` which are known HTTP headers, and could be configured to harden
+the security of an application communicating with the Vault endpoints. Note that vulnerability
+scans often examine such security related HTTP headers. In addition, an Application specific
+custom headers can also be configured. For example, `"X-Custom-Header"` has been configured
+in the example below.
+
+```hcl
```

Comment:
Would it make sense to document the order of precedence of the header values both in the configuration and as it pertains to the headers set in https://www.vaultproject.io/api/system/config-ui (or perhaps a note in config-ui), or do you think it's implied from the example here?

Reply:
sounds good!